pkg/sql/export.go

```diff
@@ -196,6 +200,10 @@ func buildExportPlanningInfo(
 	exprEval := planner.ExprEvaluator("EXPORT")
 	treeOptions := make(tree.KVOptions, len(options))
 	for i, o := range options {
+		if o.Value.(*tree.CastExpr).Expr == tree.DNull {
+			o.Value = nil
```

Comment:
i'd prefer not to mutate `options` here. can you do this in another way that doesnt? maybe setting nil on `treeOptions[i]`